azure-pipelines.yml

```diff
@@ -53,10 +53,44 @@ steps:
     testResultsFiles: '**/*.trx'
     failTaskOnFailedTests: true
   condition: succeededOrFailed()
+- task: EsrpCodeSigning@1
```

Comment:
Should this be part of the release pipeline instead, or conditionally sign builds only from master / v3.x branch or build text in subject [Build MSI]? That would avoid generating and signing MSI for every commit.

Reply:
Done; moved to release pipeline.